frontend/packages/dev-console/src/components/edit-deployment/utils/edit-deployment-utils.ts

```diff
@@ -1,4 +1,25 @@
-export const getContainerNames = (containers: { [key: string]: any }[]) => {
+import * as _ from 'lodash';
+import { ContainerSpec, EnvVar, K8sResourceKind } from '@console/internal/module/k8s';
```

Comment:
Can we go with `DeploymentKind` instead of `K8sResourceKind`?

Reply:
Same as above.